site/content/docs/03-run-time.md

```diff
@@ -156,6 +156,8 @@ Like lifecycle functions, this must be called during component initialisation.
 </script>
 ```
 
+> Context is not inherently reactive, if you need reactive values in context then you can pass a store into context, which *will* be reactive.
```

Comment:
That comma splice tho

Reply:
not, sure, what, you, mean.

Reply:
"In English grammar, a comma splice or comma fault is the use of a comma to join two independent clauses. For example: It is nearly half past five, we cannot reach town before dark."

Reply:
(In the other place you added this in the docs, you made it two completely separate sentences, which seems fine.)

Reply:
That make sense. I thought it looked a bit weird. I've changed it to match the other one.